css/custom.css

```diff
@@ -7,6 +7,26 @@ body {
 * {
     box-sizing: border-box;
 }
+.toTop{
+background-color: #00b3b3;
```

Comment:
styles ==> alphabetical sort and check indentation

Reply:
yes sir